uhabits-android/src/main/java/org/isoron/uhabits/receivers/UpdateReceiver.kt

```diff
@@ -0,0 +1,14 @@
+package org.isoron.uhabits.receivers
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+
+class UpdateReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context, intent: Intent) {
+        // Dummy receiver, relevant code is executed through HabitsApplication.
+        Log.d("UpdateReceiver", "Update receiver called.")
+    }
+}
```

Comment:
Why do we need a new (empty) receiver? Could we add `MY_PACKAGE_REPLACED` to the list of intent filters of ReminderReceiver instead?

Reply:
See comment above.